### ISSUE TITLE
Can not start GSconnect: wrong file permissions

### ISSUE THREAD

**Issue by doubleloop** [andyholmes/gnome-shell-extension-gsconnect#601]
## Log:
```
gnome-shell[3033]: JS ERROR: Gio.DBusError: Failed to execute program org.gnome.Shell.Extensions.GSConnect: Permission denied
	_GetManagedObjects/</<@/home/doubleloop/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/shell/remote.js:425:39
```

## System Details

 - **GSConnect version:**  c2bfd915
   - **Installed from:** GitHub
 - **GNOME/Shell version:** 3.30
 - **Distro/Release:** Debian testing

**Comment by terencode** [comment]
This should already be fixed with the latest commits.

**Comment by doubleloop** [comment]
>This should already be fixed with the latest commits.

c2bfd91 is current master

**Comment by andyholmes** [comment]
fixed in 64a53400

**Comment by terencode** [comment]
@doubleloop my bad, it's just that the exact same problem happened before already.
